src/@components/MainPage/hooks/useBannerSwiper.ts

```diff
@@ -0,0 +1,10 @@
+import { SwiperOptions } from "swiper";
+
+export default function useBannerSwiper() {
+  const swiperSettings: SwiperOptions = {
+    slidesPerView: 1,
+    loop: true,
+  };
+
+  return { swiperSettings };
+}

```

Comment:
이건 커스텀 훅이 되지 않아요0 그냥 객체일 뿐이에요.

```suggestion
export const swiperSettings: SwiperOptions = {
  slidesPerView: 1,
  loop: true,
};
```

Hook은 useState, useEffect와 같이 라이프사이클에 기반하여 React 내부에서 감지되고, 동작해요.
그래서 Custom Hook은 이런 `use~` 가 내부에 있을 때 구성하는 훅이에요

(useState를 내부에 넣고 함수명을 `bannerSwiper`로 지었다? 하면 lint에서 에러가 나요. `use~` 접두어를 붙여줘! 하고요)

지금은 어차피 state나 recoilState를 넣을 것이니 커스텀 훅은 그대로 놓아도 괜찮습니다!

Reply:
오오오오옷 이해했습니다😆👍 친절한 설명 감사합니당🌸🌸